app/players/elves/druid.py

```diff
@@ -0,0 +1,17 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(self, nickname: str,
+                 musical_instrument: str,
+                 favourite_spell: str) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._favourite_spell = favourite_spell
+
+    def get_rating(self) -> int:
+        return len(self._favourite_spell)
+
+    def player_info(self) -> str:
+        return (f"Druid {self.nickname}. "

```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one, the closing bracket is also written on a new line